### PR TITLE
Add new translation on Documentation

### PR DESCRIPTION
**Pull request by 0420syj**
I've translated homepage(content/_index.ko.md) and sidebar menu.
Also edited two toml files(config.toml, themes/hugo-theme-learn/i18n/ko.toml)

I've tested this files on my local server with 'Hugo'. It successfully worked.
And I've signed the CLA a few weeks ago.

Thanks for your kind help from my issue statement. It really helped me how to do all these works.

### CODE REVIEW

File: docs_source_files/config.toml [hunk 1]
```diff
@@ -224,3 +224,35 @@ weight = 20
 name = "<i class='fas fa-fw fa-bullhorn'></i> 貢献方法"
 url = "contributing"
 weight = 30
+
+[Languages.ko]
+title = "Selenium 문서"
+weight = 6
```
Comment (diemol) [inline docs_source_files/config.toml]: Language `ja` has already weight 6, can you please change it to 7?

File: docs_source_files/config.toml [hunk 2]
```diff
@@ -224,3 +224,35 @@ weight = 20
...
+
+[[Languages.ko.menu.shortcuts]]
+name = "<i class='fas fa-home'></i> Selenium 홈"
+url = "https://selenium.dev/"
+weight = 1
+
+[[Languages.ko.menu.shortcuts]]
+name = "<i class='fab fa-fw fa-github'></i> GitHub repo"
+identifier = "ds"
+url = "https://github.com/SeleniumHQ/site"
```
Comment (diemol) [inline docs_source_files/config.toml]:
```suggestion
url = "https://github.com/SeleniumHQ/seleniumhq.github.io"
```

File: docs_source_files/config.toml [hunk 3]
```diff
@@ -224,3 +224,35 @@ weight = 20
...
+weight = 10
+
+[[Languages.ko.menu.shortcuts]]
+name = "<i class='fas fa-fw fa-exclamation-triangle'></i> 버그 신고"
+url = "https://github.com/seleniumhq/site/issues"
```
Comment (diemol) [inline docs_source_files/config.toml]:
```suggestion
url = "https://github.com/seleniumhq/seleniumhq.github.io/issues"
```

File: docs_source_files/config.toml (hunk 1 above)
Comment (0420syj) [reply docs_source_files/config.toml]: Thanks. changed it to 7

File: docs_source_files/config.toml (hunk 2 above)
Comment (0420syj) [reply docs_source_files/config.toml]: sorry for my mistake. thanks.

File: docs_source_files/config.toml (hunk 3 above)
Comment (0420syj) [reply docs_source_files/config.toml]: Done!

**Pull request merged**